net-irc/znc/znc-1.6.3.ebuild

```diff
@@ -0,0 +1,180 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{3_3,3_4,3_5} )
+inherit eutils python-single-r1 systemd user
+
+MY_PV=${PV/_/-}
+GTEST_VER="1.7.0"
+GTEST_URL="https://googletest.googlecode.com/files/gtest-${GTEST_VER}.zip"
+DESCRIPTION="An advanced IRC Bouncer"
+
+SRC_URI="http://znc.in/releases/${PN}-${MY_PV}.tar.gz
+	test? ( ${GTEST_URL} )"
+KEYWORDS="~amd64 ~arm ~x86"
+
+HOMEPAGE="http://znc.in"
+LICENSE="Apache-2.0"
+SLOT="0"
+IUSE="daemon debug ipv6 libressl perl python ssl sasl tcl test"
+
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+RDEPEND="
+	dev-libs/icu
+	sys-libs/zlib
+	perl? ( >=dev-lang/perl-5.10 )
+	python? ( ${PYTHON_DEPS} )
+	sasl? ( >=dev-libs/cyrus-sasl-2 )
+	ssl? (
+		!libressl? ( dev-libs/openssl:0 )
+		libressl? ( dev-libs/libressl )
+	)
+	tcl? ( dev-lang/tcl:0= )
+"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig
+"
+
+S=${WORKDIR}/${PN}-${MY_PV}
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-1.6.1-systemwideconfig.patch
+	"${FILESDIR}"/${PN}-1.6.1-create-pidfile-per-default.patch
+	"${FILESDIR}"/${PN}-1.6.3-libressl.patch
+)
+
+ZNC_DATADIR="${ZNC_DATADIR:-"/var/lib/znc"}"
+
+pkg_setup() {
+	if use python; then
+		python-single-r1_pkg_setup
+	fi
+	if use daemon; then
+		enewgroup ${PN}
+		enewuser ${PN} -1 -1 /dev/null ${PN}
+	fi
+}
+
+src_unpack() {
+	default
+
+	if use test; then
+		cd "${S}"/test || die "Failed to chdir into '${S}/test'"
+		unpack ${GTEST_URL##*/}
```

Comment:
This should already be unpacked by the default command but probably not where you expect it to be. So all you should have to do is move it into the right place.